apps/portals/src/configurations/arkportal/style/_style_overrides.scss

```diff
@@ -113,7 +113,7 @@ $ARK-gray-900: #1d1d22 !default;
       line-height: 1.5;
       color: $ARK-gray-700;
       margin-bottom: 40px;
-      &__link.btn.pill {
+      &__link.btn {
```

Comment:
Maintain custom styling of Goal Cards on ARK Page: 
![feature_ARK_About_Desktop](https://user-images.githubusercontent.com/26949006/231903743-6bf5da88-370b-4474-bea0-1cfda070ab13.png)